### PR TITLE
initial changes

### PR DESCRIPTION
**Pull request by dangrayza**
Hi Steve, I made a few small changes.

### CODE REVIEW

File: src/app/pages/missions-page/missions-page.component.html [hunk 1]
```diff
@@ -14,11 +14,39 @@ <h3 id="about-us-title" class="text-left">
...
 
                 <p class="text-left">
-                    Synergy in Mission (SIM) represents the various ministries and initiatives that New Creation Family Church is serving in God’s mission both locally and around the world. 
-                    Synergy describes the ongoing collaboration toward the completion of Christ’s Great Commission. 
-                    The SIM Team meets regularly for prayer, planning and action. Sharon Stephen serves and the SIM Coordinator. 
-                    <br><br>
-                    Please contact <a href="mailto:admin@newcreation.co.za">admin@newcreation.co.za</a> to find out more about the SIM vision, ministries and how you can get more involved. 
+                    Synergy in Mission (SIM) represents the various ministries and initiatives that New Creation Family Church is serving in God’s mission both locally and around the world.
+                    Synergy describes the ongoing collaboration toward the completion of Christ’s Great Commission.
+                    The SIM Team meets regularly for prayer, planning and action. Sharon Stephen serves and the SIM Coordinator.
```
Comment (steichert) [inline src/app/pages/missions-page/missions-page.component.html]: Just a small typo correction: Sharon Stephen serves **as** the SIM Coordinator

File: src/app/pages/missions-page/missions-page.component.html [hunk 2]
```diff
@@ -14,11 +14,39 @@ <h3 id="about-us-title" class="text-left">
...
+                    <li>Sharing the Gospel of Jesus Christ with unbelievers Local and Beyond</li>
+                    <li>Partnering with individuals and Organisations that align with the Great Commission</li>
+                    <li>Inspire other Christians to join the call to be partners in the Great Commission</li>
+                    <li>Bearing fruit through the power of the Holy Spirit</li>
+                </ol>
+
+
+                <p class="text-left"><b>How do we do this?</b></p>
+                <ol>
+                    <li>Seeking opportunities to for personal witnessing</li>
```
Comment (steichert) [inline src/app/pages/missions-page/missions-page.component.html]: Another typo correction: Seeking opportunities **for** personal witnessing

File: src/app/resources/missions-screen-constants.ts [hunk 3]
```diff
@@ -1,51 +1,59 @@
...
-        cardTitle: "CIA: Community In Action",
+        cardTitle: "Gardens of Faithfulness",
+        // cardTitleColour: "#212529",
         cardTitleColour: "white",
         description: "",
-        cardImageUrl: 'https://res.cloudinary.com/dbmlnkfvh/image/upload/v1673438138/static/missions/cia-card_fvakrl.jpg',
-        mainImageUrl: 'https://res.cloudinary.com/dbmlnkfvh/image/upload/v1673438047/static/missions/cia_zm417w.jpg',
-        mainTitle: "CIA: Community in Action",
+        cardImageUrl: `${environment.imageUrl}/missions/gardens.jpg`,
+        mainImageUrl: `${environment.imageUrl}/missions/gardens.jpg`,
```
Comment (steichert) [inline src/app/resources/missions-screen-constants.ts]: I've uploaded this image to the Cloudinary image hosting service for you. You can use this link for the image instead: https://res.cloudinary.com/dbmlnkfvh/image/upload/v1706774631/gardens_dxzzyo.jpg

**Review by steichert — COMMENTED** [review]
Thanks, @dangrayza. The changes look great! Just added a few minor comments.

**Comment by dangrayza** [comment]
Thanks, @steichert! I've made the changes.

**Comment by steichert** [comment]
> Thanks, @steichert! I've made the changes.

Thanks @dangrayza, looks great!

**Review by steichert — APPROVED** [review]

**Pull request merged**